go.mod

```diff
@@ -58,26 +57,18 @@ require (
 	github.com/beorn7/perks v1.0.1 // indirect
 	github.com/cespare/xxhash/v2 v2.1.2 // indirect
 	github.com/dgryski/go-rendezvous v0.0.0-20200823014737-9f7001d12a5f // indirect
-	github.com/facebookgo/clock v0.0.0-20150410010913-600d898af40a // indirect
-	github.com/facebookgo/limitgroup v0.0.0-20150612190941-6abd8d71ec01 // indirect
-	github.com/facebookgo/muster v0.0.0-20150708232844-fd3d7953fd52 // indirect
-	github.com/felixge/httpsnoop v1.0.3 // indirect
 	github.com/golang/protobuf v1.5.2 // indirect
 	github.com/golang/snappy v0.0.0-20180518054509-2e65f85255db // indirect
-	github.com/honeycombio/libhoney-go v1.16.0 // indirect
-	github.com/klauspost/compress v1.15.7 // indirect
 	github.com/matttproud/golang_protobuf_extensions v1.0.1 // indirect
 	github.com/pelletier/go-toml v1.9.3 // indirect
 	github.com/prometheus/common v0.37.0 // indirect
 	github.com/prometheus/procfs v0.8.0 // indirect
+	github.com/rogpeppe/go-internal v1.8.0 // indirect
```

Comment:
Why is this showing up as a new dependency? I don't see anything in the rest of the PR that introduces this, and I don't see any new files showing up in //vendor.

Reply:
I'm not entirely sure!  "go mod tidy" is quite insistent that it should be here though.  I'm going to try pushing a commit without it and see what happens in CI.

Reply:
Well, CI seems happy without it, so I'm pretty confused.  I think I'm happy to write this off as a mystery for now.